sources/packages/web/src/views/institution/Login.vue

```diff
@@ -35,15 +33,20 @@
 </template>
 
 <script lang="ts">
+import { ref } from "vue";
 import { AppConfigService } from "../../services/AppConfigService";
-import { useRoute } from "vue-router";
 
 export default {
   components: {},
-
+  props: {
+    showBasicBCeIDMessage: {
+      type: Boolean,
+      required: true,
+      default: false,
+    },
+  },
   setup(props: any) {
-    const route = useRoute();
-    const basicBCeID = route.query.basicBCeID;
+    const basicBCeID = ref(props.showBasicBCeIDMessage);
```

Comment:
I would recommend accessing directly the showBasicBCeIDMessage on Vue template instead of creating a new basicBCeID based on the property.

Reply:
I tried that vue compiler was throwing some error, I will retry ...

Reply:
NOT WORKING